addons/account/i18n/account.pot

```diff
@@ -2418,7 +2418,7 @@ msgstr ""
 #. module: account
 #: model:ir.model.fields,help:account.field_account_fiscal_position__auto_apply
 #: model:ir.model.fields,help:account.field_account_fiscal_position_template__auto_apply
-msgid "Apply automatically this fiscal position."
+msgid "Apply tax & account mappings on invoices automatically if the matching criterias (VAT/Country) are met."
```

Comment:
Actually, I didn't think about it earlier, but you don't need to change the .pot file too as it is targeting master.
In master branch, the .pot files are regularly updated so you don't need to do it yourself (although it's not bad to do it I guess)

Reply:
Alright thank you very much for your review and for the lesson. I'm therefore going to update the file